wordpress-mobile/WordPress-Android
# Issue/editor button long press

### Fix
Remove the pressed state color from format toolbar buttons as done in https://github.com/wordpress-mobile/AztecEditor-Android/pull/346.

### Test
0. Enable ***Beta*** editor.
1. Go to ***Sites*** tab.
2. Tap ***Create*** floating button.
3. Tap content text field.
4. Long-press any format toolbar button.
5. Notice color remains unchanged.
6. Notice ripple effect.
